### PR TITLE
Improves the ahelp resolution message

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Improves the adminhelp resolution message:
 - It now is a large message in chat, indicating that the adminhelp has been resolved.
 - The message now contains text saying that appropriate action has been taken if required (Users don't know what was taken, but should know that action *may* have been taken even if they don't know it).
 - The ckey of the handling administrator is now included in the resolution box, making it easier to know who handled a ticket (Useful for following up handling of bad tickets).

## Why It's Good For The Game

Players will be informed that action has been taken if required, they should know that even if they aren't explicitly told what happened to the other person, something did happen.
Better ability for players to follow up on poor ticket handling, admins that are handling tickets improperly can be trained and talked to.

## Testing Photographs and Procedure

image

image

image

image

image

## Changelog
:cl:
admin: The adminhelp resolution message has been updated.
/:cl:

### CODE REVIEW

**Comment by Fronsis** [comment]
Admemes still have the chance to hide their ckey right? Or with this it's gonna go away and enforce clarity? I remember some ahelps i've done in which the admin was: Admin which i reckon it's from when they're in hidden mode, i'd like for the clarity thing tbh

**Comment by PowerfulBacon** [comment]
Once a ticket is resolved, the real ckey will display even in stealth mode

File: code/modules/admin/verbs/adminhelp.dm [hunk 1]
```diff
@@ -615,17 +615,23 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 		discordsendmsg("ahelp", "Ticket #[id] closed by [key_name(usr, include_link=0)]")
 
 //Mark open ticket as resolved/legitimate, returns ahelp verb
-/datum/admin_help/proc/Resolve(key_name = key_name_admin(usr), silent = FALSE)
+/datum/admin_help/proc/Resolve(key_name = key_name_admin(usr), silent = FALSE, status = "Resolved", message = null)
 	if(state > AHELP_ACTIVE)
 		return
+
+	var/final_output = "<span class='adminhelp_conclussion'><span class='big'><b>Adminhelp [status]</b></span><br />"
```
Comment (itsmeow) [inline code/modules/admin/verbs/adminhelp.dm]: conclussion?

File: tgui/packages/tgui-panel/styles/goon/chat-dark.scss [hunk 2]
```diff
@@ -323,6 +323,21 @@ em						{font-style: normal;	font-weight: bold;}
...
+  display: block;
+  color: white;
+  text-align: center;
+  background-color: black;
+  border: solid;
+  border-color: red;
+  border-width: 2px;
+  border-radius: 10px;
+  padding: 10px;
+  margin-left: 20px;
```
Comment (itsmeow) [inline tgui/packages/tgui-panel/styles/goon/chat-dark.scss]: `margin: 10px 20px` is more concise

File: tgui/packages/tgui-panel/styles/goon/chat-dark.scss [hunk 3]
```diff
@@ -323,6 +323,21 @@ em						{font-style: normal;	font-weight: bold;}
 .boldnotice				{color: #6685f5;	font-weight: bold;}
 .adminnotice			{color: #6685f5;}
 .adminhelp              {color: #ff0000;    font-weight: bold;}
+.adminhelp_conclussion {
+  display: block;
+  color: white;
+  text-align: center;
+  background-color: black;
+  border: solid;
```
Comment (itsmeow) [inline tgui/packages/tgui-panel/styles/goon/chat-dark.scss]: `border: 2px solid red`

**Review by itsmeow — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/adminhelp.dm (hunk 1 above)
Comment (PowerfulBacon) [reply code/modules/admin/verbs/adminhelp.dm]: Crap, it's spelt wrong isn't it

**Comment by ro5490** [comment]
Might be a good idea to append a line to the ahelp resolved message that links to the Admin Feedback area(or specific admin thread) on the forums, as feedback posts dont get enough love really

File: tgui/packages/tgui-panel/styles/goon/chat-dark.scss (hunk 2 above)
Comment (PowerfulBacon) [reply tgui/packages/tgui-panel/styles/goon/chat-dark.scss]: Its less clear but whatever idc

**Review by itsmeow — APPROVED** [review]

**Pull request merged**